src/tensor_parallel/slicer_wrapper.py

```diff
@@ -88,16 +89,33 @@ def get_default_config(cls, module: nn.Module) -> Config:
                 assert module.weight.shape == shape, f"{module.weight.shape} != {shape}"
                 assert module.bias is None or module.bias.shape == (module.out_channels,), module.bias.shape
                 config.state_rules[f"^{name}.weight$"] = "split 1" if module.transposed else "split 0"
-                config.state_rules[f"^{name}.bias$"] = "split 0"
+                if module.bias is not None:
+                    config.state_rules[f"^{name}.bias$"] = "split 0"
                 config.output_rules[f"^{name}$"] = {0: "gather 1"}
             elif isinstance(module, conv._ConvNd) and module.groups != 1:
-                logger.warning(
-                    f"AutoConfig does not support sharding convolution layers {name} with {module.groups} groups yet; "
-                    "If you're sure that you need it, implement sharding in place of this message and open a PR"
-                )
+                # group conv: split each group individually over input channels to avoid changing module.groups
```

Comment:
Why is it important to not change num_groups? if num_groups % world_size != 0, ranks would get an uneven number of groups which would require a custom convolution operation. If user wants this behavior for their specific model, we humbly recommend them to write their own config :)